models/board.py

```diff
@@ -0,0 +1,75 @@
+import datetime
+from peewee import Model, CharField, IntegrityError, DateTimeField
+
+from . import DB
```

Comment:
라이브러리가 아닌 일반 어플리케이션에서 relative import는 편하긴 하지만 오히려 독이 될 수 있습니다.
```suggestion
from models import DB
```